src/dsp_tools/xmllib/value_converters.py

```diff
@@ -18,6 +18,24 @@ def convert_to_bool_string(value: Any) -> str:
     Returns:
         `"true"` or `"false"` if it is an accepted value,
         else it returns the original value as a string.
+
+    Examples:
+        ```python
+        result = xmllib.convert_to_bool_string(1)
+        # -> "true"
+        ```
+
+        ```python
+        result = xmllib.convert_to_bool_string("nein")
+        # -> "false"
+        ```
+
+        ```python
+        # because this is not an accepted value, it is returned as a string
```

Comment:
Not part of this PR, but here it stuck me that this behaviour is contra-intuitive.
There mere fact that you had to write this explanation shows that it is not natural.